.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -13,17 +13,13 @@ name: LTE integ test containerized AGW
 
 on:
   workflow_dispatch: null
-  workflow_run:
-    workflows:
-      - build-all
+  push:
```

Comment:
For this workflow it is not correct to replace the trigger by a push trigger, as it is relying on artifacts build in the build-all job.

Reply:
Could you please point me to the place where the dependency on build-all artifacts is?

Reply:
Follow:
- https://github.com/magma/magma/blob/05a2fb224aad57cf56eef159b4650d78ebe53c21/.github/workflows/lte-integ-test-containerized.yml#L74
- https://github.com/magma/magma/blob/05a2fb224aad57cf56eef159b4650d78ebe53c21/lte/gateway/fabfile.py#L510
- https://github.com/magma/magma/blob/master/lte/gateway/fabfile.py#L534
- https://github.com/magma/magma/blob/master/lte/gateway/docker/docker-compose.yaml#L45-L52
- https://github.com/magma/magma/blob/master/lte/gateway/docker/.env#L16